modules/user_accounts/ajax/rejectUser.php

```diff
@@ -58,6 +61,5 @@
     );
 }
 
-(\Database::singleton())->delete('users', array("UserID" => $username));
-header("HTTP/1.1 204 No Content");
-
+(\NDB_Factory::singleton()->database())->delete('users', array("UserID" => $username));
```

Comment:
```suggestion
($factory->database())->delete('users', array("UserID" => $username));
```

Since you declared it above. No big deal.